src/components/tasks/TaskList.vue

```diff
@@ -40,4 +31,10 @@ export default {
   border-top-right-radius: 5px;
   box-shadow: 0px 38px 12px 0px rgb(120, 120, 120);
 }
+
+.noTasks {
+  padding: 10px;
+  text-align: center;
+  color: #b0b0b0;
```

Comment:
I noticed that there are colors that are repeated in the styles of different components. I think it makes sense to make global variables for common colors and use them in components